.spectral.yaml

```diff
@@ -1,6 +1,11 @@
-extends: '@rhoas/spectral-ruleset'
+extends: 'bin/rhoasapi-installation/node_modules/@rhoas/spectral-ruleset/ruleset.yaml'
```

Comment:
Why this change?

Reply:
newest spectral version will not locate the ruleset without being expressly told where it is. it is indeed uglier but linter will not find the ruleset otherwise

Reply:
:-( this sounds like a bug more than a feature

Reply:
I genuinely do not know what the reason is for this. it appears to be just this repo that has the issue. want me to create a link to it instead?

Reply:
Yes. A link to the issue will be fine. Thank you so much.